lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,336 @@
+            child: const SizedBox.expand(
+              child: ColoredBox(
+                color: Colors.white30,
+                child: Center(child: CircularProgressIndicator()),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create the map.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISOceans);
+
+    // Create a point using x and y coordinates.
+    final point = ArcGISPoint(
+      x: 56.075844,
+      y: -2.681572,
+      spatialReference: SpatialReference.wgs84,
+    );
+
+    // Set the initial viewpoint of the map to the point and provide a scale.
+    map.initialViewpoint = Viewpoint.fromCenter(point, scale: 2000);
+
+    // Set the map to the mapview controller.
+    _mapViewController.arcGISMap = map;
+
+    // Add graphics overlay to the mapview controller.
+    _mapViewController.graphicsOverlays.add(_graphicsOverlay);
+
+    // Call functions to create the graphics.
+    _createPoints();
+    _createPolygon();
+    _createPolyline();
+    _createText();
+
+    // Update the extent to encompass all of the symbols.
+    _setExtent();
+
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  void _createPoints() {
+    // Create a red circle simple marker symbol.
+    SimpleMarkerSymbol redCircleSymbol = SimpleMarkerSymbol(
+        style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 10,);
+
+    // Create graphics and add them to graphics overlay.
+    Graphic graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.72,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.69,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.66,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.63,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createPolyline() async {
+    // Create a purple simple line symbol.
+    SimpleLineSymbol lineSymbol = SimpleLineSymbol(
+      style: SimpleLineSymbolStyle.dash,
+      color: Colors.purple,
+      width: 10,
+    );
+
+    // Build a polyline.
+    final polylineBuilder =
+        PolylineBuilder(spatialReference: SpatialReference.wgs84);
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.715,
+        y: 56.061,
+      ),
+    );
```

Comment:
There's a more compact way to do this. Instead of `addPoint`, use `addPointXY`
```dart
polylineBuilder.addPointXY(x: -2.715, y:56.061);
```
That'll save a lot of bulk here and in `_createPolygon` too.